Source/WebCore/platform/cocoa/WebAVPlayerLayer.mm

```diff
@@ -300,6 +300,10 @@ - (void)resolveBounds
     [_videoSublayer setFrame:_targetVideoFrame];
 
     [CATransaction commit];
+
+#if HAVE(UI_WINDOW_SCENE_LIVE_RESIZE)
+    _fullscreenModel.get()->didResolvePlayerLayerBounds();
```

Comment:
```suggestion
    [_fullscreenModel didResolvePlayerLayerBounds()];
```

probably

Reply:
Does that work? `_fullscreenModel` is not an objective-c type I think

Reply:
oh, ya that's probably why I didn't do it in the first place

Reply:
But `.get()` is unnecessary. `_fullscreenModel->didResolvePlayerLayerBounds();` should work.

Reply:
I changed it to

```
    if (auto model = _fullscreenModel.get())
        model->didResolvePlayerLayerBounds();
```

to match the rest of the file